src/CabanaPD_HeatTransfer.hpp

```diff
@@ -0,0 +1,101 @@
+/****************************************************************************
+ * Copyright (c) 2022-2023 by Oak Ridge National Laboratory                 *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of CabanaPD. CabanaPD is distributed under a           *
+ * BSD 3-clause license. For the licensing terms see the LICENSE file in    *
+ * the top-level directory.                                                 *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef HEATTRANSFER_H
+#define HEATTRANSFER_H
+
+#include <CabanaPD_Timer.hpp>
+
+namespace CabanaPD
+{
+
+// Peridynamic heat transfer with forward-Euler time integration.
+// Inherits only because this is a similar neighbor-based kernel.
+template <class ExecutionSpace, class ModelType>
+class HeatTransfer : public Force<ExecutionSpace, BaseForceModel>
+{
+  protected:
+    using base_type = Force<ExecutionSpace, BaseForceModel>;
+    using base_type::_half_neigh;
+    using base_type::_timer;
+    ModelType _model;
+
+  public:
+    HeatTransfer()
+        : base_type( false ){};
+
+    HeatTransfer( const bool half_neigh, const ModelType model )
+        : base_type( half_neigh )
+        , _model( model )
+    {
+    }
+
+    template <class TemperatureType, class PosType, class ParticleType,
+              class NeighListType, class ParallelType>
+    void
+    computeHeatTransferFull( TemperatureType& temp, const PosType& x,
```

Comment:
I assume this mimics "ComputeForceFull"? Do we have "ComputeForce" also? If so, where?

Reply:
I don't understand - are you asking about `ComputeForce` as distinct from `ComputeForceFull` in the PMB/LPS files? What capability are you asking about?

Reply:
`TemperatureType` is the type for the temperature data we pass here, same as all other particle-field template types

Reply:
Sorry, I found ""ComputeForce" in CabanaPD_Force.hpp.